### ISSUE TITLE
`latest-passing-build` tag is not updated

### ISSUE THREAD

**Issue by thetic** [cpputest/cpputest#1637]
I'm guessing this used to be set by the travis automation?

**Comment by asimov-aiz** [comment]
+1 to this. I got several failure on latest-passing-build but passing on master

**Comment by basvodde** [comment]
@thetic Yes, it was. Also was it making a zip file to upload to github. The latter has been broken for a while due to changes in the APIs. Since now Travis is defunct, it is even more broken.

**Comment by thetic** [comment]
OK, I'm going to try to translate whatever is missing from the old travis config into GitHub actions.
